Dockerfile.rhel8

```diff
@@ -30,6 +30,9 @@ WORKDIR $HIVE_HOME
 
 ENV HADOOP_CLASSPATH $HIVE_HOME/hcatalog/share/hcatalog/*:${HADOOP_CLASSPATH}
 
+# make JNDI impossible from log4j usages
+RUN zip -q -d lib/log4j-core-*.jar org/apache/logging/log4j/core/lookup/JndiLookup.class
```

Comment:
Just a quick note for our future selves: https://github.com/kube-reporting/hive/pull/68 exposed that the hadoop base image doesn't have zip present.